joomla/joomla-cms
# [4.x][build tools] [NO CACHE]  Use the embeded sass

Pull Request is a redo of https://github.com/joomla/joomla-cms/pull/33170

### Summary of Changes
- Change the SASS package
- perf gains (less time for compilation of sass) read more here: https://sass-lang.com/blog/embedded-sass-is-live

### Testing Instructions
Before applying this PR run `npm install` on a clean git fork and move the media folder with another name (eg media_old)
Apply this PR and run again `nom install` 
compare the folders `media` and `media_old` should be the same 


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required
